pages/common/zsh.md

```diff
@@ -1,7 +1,7 @@
 # zsh
 
-> Z SHell.
-> `bash` and `sh`-compatible command line interpreter.
+> Z SHell, a `bash` and `sh`-compatible command line interpreter.
+> Please refer to `tldr histexpand` for history expansion.
```

Comment:
Perhaps this will read better?
```suggestion
> Z SHell, a `bash` and `sh`-compatible command line interpreter.
> See also `histexpand` for history expansion.
```